src/errors/argument.error.ts

```diff
@@ -0,0 +1,8 @@
+export class ArgumentError extends Error {
```

Comment:
Should we create a Base error class like we did with FakerError (e.g. CliError)?

Reply:
That would not have any advantage unless we want to handle all CLI errors in a specific way (ex apply prefix or global "style"). In Faker we did it so a user could potentially narrow down an error in their code:

```ts
try {
  // do something and call some faker method
} catch(error: unkown) {
  if(error instanceof FakerError) {
    // now they know Faker threw the error
  } else {
    // something else threw
  }
}
``` 

Since a CLI doesn't have this usecase, it don't see the necessity.